beacon_node/execution_layer/src/engine_api/json_structures.rs

```diff
@@ -363,27 +326,20 @@ impl<T: EthSpec> From<ExecutionPayload<T>> for JsonExecutionPayload<T> {
                 block_hash: capella.block_hash,
                 transactions: capella.transactions,
                 #[cfg(feature = "withdrawals")]
-                withdrawals: capella.withdrawals,
-            }),
-            ExecutionPayload::Eip4844(eip4844) => Self::V3(JsonExecutionPayloadV3 {
-                parent_hash: eip4844.parent_hash,
-                fee_recipient: eip4844.fee_recipient,
-                state_root: eip4844.state_root,
-                receipts_root: eip4844.receipts_root,
-                logs_bloom: eip4844.logs_bloom,
-                prev_randao: eip4844.prev_randao,
-                block_number: eip4844.block_number,
-                gas_limit: eip4844.gas_limit,
-                gas_used: eip4844.gas_used,
-                timestamp: eip4844.timestamp,
-                extra_data: eip4844.extra_data,
-                base_fee_per_gas: eip4844.base_fee_per_gas,
-                excess_blobs: eip4844.excess_blobs,
-                block_hash: eip4844.block_hash,
-                transactions: eip4844.transactions,
-                #[cfg(feature = "withdrawals")]
-                withdrawals: eip4844.withdrawals,
+                withdrawals: Some(
+                    Into::<Vec<_>>::into(capella.withdrawals)
```

Comment:
```suggestion
                    Vec::from(capella.withdrawals)
```

Reply:
Does `Vec<T>` implement `From<VariableList<T>>`?

Reply:
yes: https://github.com/sigp/lighthouse/blob/6d5a2b509fac7b6ffe693866f58ba49989f946d7/consensus/ssz_types/src/variable_list.rs#L125-L129

`From` and `Into` are _meant_ to be mirrored (we do break from that sometimes)

Reply:
Gotcha, thanks! I knew they were inverses but I get mixed up about the **inverse operation**
 - if `A` implements `From<B>` then `B` implements `Into<A>` (same operation)
 - This does not necessarily mean `B` implements `From<A>` or (equivalently) `A` implements `Into<B>` (inverse operation)